ibis/pandas/execution.py

```diff
@@ -135,6 +138,34 @@ def execute_cast_series_date(op, data, type, scope=None):
 }
 
 
+@execute_node.register(ops.UnaryOp, pd.Series)
+def execute_series_unary_op(op, data, scope=None):
+    function = getattr(np, type(op).__name__.lower())
+    if data.dtype == np.dtype(np.object_):
+        return data.apply(functools.partial(execute_node, op, scope=scope))
```

Comment:
So, it looks like `Series.map` accepts a `dict`, to support a simple `CASE`-statement-like operation, as well as callables, whereas apply only deals with callables. For callables, both methods use the same underlying function `lib.map_infer` to call the passed in callable in a Cython loop. Here we could use either since we're only dealing with callables.